var/spack/repos/builtin/packages/turbine/package.py

```diff
@@ -16,16 +16,19 @@ class Turbine(AutotoolsPackage):
     configure_directory = 'turbine/code'
 
     version('develop', branch='master')
-    version('1.2.3', 'f2e393c292c4248b4e77a19f8272ae88')
+    version('1.2.3', '028f6f4f5041e5ffbbf3ab8008e3b781')
```

Comment:
Any idea why this hash changed? We don't want a repeat of https://github.com/Homebrew/homebrew-cask/issues/33536

Reply:
This was an intentional fix, the hash is correct.